sycl/include/CL/sycl/detail/array.hpp

```diff
@@ -84,7 +84,7 @@ template <int dimensions = 1> class array {
   array<dimensions> &operator=(const array<dimensions> &rhs) = default;
   array<dimensions> &operator=(array<dimensions> &&rhs) = default;
 
-  // Returns true iff all elements in 'this' are equal to
+  // Returns true if all elements in 'this' are equal to
```

Comment:
Do you really mean to change these?  'iff' typically means "if and only if", so I presume it is intentional here.

https://en.wikipedia.org/wiki/If_and_only_if

Reply:
Ok, thanks!